metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/CloudWatchRecorder.java

```diff
@@ -117,11 +181,16 @@
      *
      * @param namespace CloudWatch namespace to publish under.
      * @return A CloudWatchRecorder instance that will automatically shutdown on JVM exit.
+     * @deprecated Use {@link CloudWatchRecorder.Builder} instead
```

Comment:
Do we need to deprecate these constructors?

While I agree that some customers will prefer the builder I also think some may prefer the constructors as well. Maintaining both can present some challenges (e.g. keeping defaults in sync) but as library vendors I'm not sure this is somewhere we need to be opinionated about how someone uses our library.

Reply:
No, they don't need to be deprecated. They would be phased out in the ideal case, but I'm not in a position to drive the deprecation campaign, so I'm fine with leaving them as-is and adding the builder.